content/en/about/community/join/index.md

```diff
@@ -45,13 +45,19 @@ Once you've done that, you can head straight to our
 {{< /community_item >}}
 
 {{< community_item logo="./group.svg" alt="Working Groups" >}}
-If you'd like to contribute to the Istio project, learn [how we do things](https://github.com/istio/community/blob/master/README.md) and
-consider participating in our
-[working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+If you'd like to contribute to the Istio project, please read our
+[code of conduct agreement](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and
```

Comment:
Looking at the preview, I think I agree with @ericvn's comment about the word "agreement". It's not necessary.
```suggestion
[code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct) and
```